rpm/totem-pl-parser.spec

```diff
@@ -1,14 +1,12 @@
 Name:       totem-pl-parser
 Summary:    Totem Playlist Parser library
-Version:    3.26.1
+Version:    3.26.6
 Release:    1
-Group:      System/Libraries
 License:    LGPLv2+
 URL:        http://www.gnome.org/projects/totem/
 Source0:    http://download.gnome.org/sources/%{name}/3.26/%{name}-%{version}.tar.bz2
```

Comment:
Extra points: This should be %{name}-%{version}.tar.bz2 as the upstream source tarball is not the one we use here directly.

Reply:
Sure. Updated.